tive um problema que nao sonseguir resolver que era
"──(root㉿kali)-[/home
└─# python encrypter.py
Traceback (most recent call last):
  File "/home/zarbaqz/ramsoware/encrypter.py", line 2, in <module>
    import pyaes
ModuleNotFoundError: No module named 'pyaes' "
por mais que eu atualiasse ou fazer intalação do pyaes dava memso erro 
